### PR TITLE
Use separate expiration date to display users their expired ads.

### PR DESCRIPTION
**Pull request by shylux**
Currently the plone default expiration date is used to remove old ads from the billboard. Those ads can only be listed by admins (or users with the specific site-wide permission). We want to give the user the ability to access their expired ads to modify/extend them.

The following tasks are required:
- [x] Add separate DateTime field
  - [x] Delete the expirationDate value from existing ads and put it in the new field. This makes them visible again.
  - [x] Add index for the new field.
  - [x] Add indexer. Because the pictures inside the ads get the indexed function over aquisition and thus get indexed themself too.
  - [x] Add metadata column for expiration checks in template.
  - [x] Change the ad listing to filter out ads based on the new field.
- [x] Add a listing which shows all adds of the logged-in user.
- [x] Update potential other views/customisations
- [x] Translations
- [x] Changelog

Extra:
- [x] Update picture overlay to wrap all pictures together in a slideshow.

screen shot 2015-01-15 at 09 19 37
screen shot 2015-01-15 at 09 19 50
screen shot 2015-01-15 at 09 20 20
screen shot 2015-01-15 at 09 20 31

### CODE REVIEW

File: ftw/billboard/locales/de/LC_MESSAGES/ftw.billboard.po [hunk 1]
```diff
@@ -71,13 +75,21 @@ msgstr "Validierung gescheitert ($name: keine gültigge E-Mail Adresse)"
...
+
 #. Default: "Please accept the <a target=\"_blank\" href=\"./billboard_conditions\">conditions<a/>."
-#: ./ftw/billboard/content/billboardad.py:87
+#: ./ftw/billboard/content/billboardad.py:98
 msgid "help_accept_conditions"
 msgstr "Bitte akzeptieren Sie die <a target=\"_blank\" href=\"./billboard_conditions\">Publikationsbedingungen<a/>."
 
+#: ./ftw/billboard/content/billboardad.py:75
+msgid "help_ad_expiration_date"
+msgstr "Die Anzeige nach diesem Datum nicht mehr angezeigt."
```
Comment (maethu) [inline ftw/billboard/locales/de/LC_MESSAGES/ftw.billboard.po]: @shylux phrasing?

**Comment by mbaechtold** [comment]
Please add a entry to the changelog.

**Comment by shylux** [comment]
@mbaechtold added changelog & fixed german sentence

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,7 +5,8 @@ CHANGELOG
 1.3.5 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added seperate expiration date.
```
Comment (mbaechtold) [inline docs/HISTORY.txt]: @shylux: there's a typo in "seperate".

Comment (shylux) [inline docs/HISTORY.txt]: @mbaechtold No spellchecker in my code editor :*(
Is fixed.

Comment (mbaechtold) [inline docs/HISTORY.txt]: No worries... Thank you!

**Comment by mbaechtold** [comment]
Looks good to me :+1:

**Pull request merged**